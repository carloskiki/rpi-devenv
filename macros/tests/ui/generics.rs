#[macros::main]
fn my_main<const N: usize, 'a, T>() {}

fn main() {}
