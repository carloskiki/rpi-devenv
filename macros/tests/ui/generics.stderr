error: main function cannot have generic parameters
 --> tests/ui/generics.rs:2:11
  |
2 | fn my_main<const N: usize, 'a, T>() {}
  |           ^^^^^^^^^^^^^^^^^^^^^^^
